wpilibj/src/athena/java/edu/wpi/first/wpilibj/ADXL362.java

```diff
@@ -49,7 +49,7 @@
     @SuppressWarnings("MemberName")
     public final byte value;
 
-    private Axes(byte value) {
+    Axes(byte value) {
```

Comment:
This is actually a change of accessibility. I don't think this is what we want here.